pkg/plugins/manager/installer/installer.go

```diff
@@ -607,12 +594,39 @@ func extractSymlink(file *zip.File, filePath string) error {
 	if _, err := io.Copy(buf, src); err != nil {
 		return fmt.Errorf("%v: %w", "failed to copy symlink contents", err)
 	}
-	if err := os.Symlink(strings.TrimSpace(buf.String()), filePath); err != nil {
+
+	symlinkPath := strings.TrimSpace(buf.String())
+	if !isSymlinkRelativeTo(basePath, symlinkPath, filePath) {
```

Comment:
This is specific to installing plugins since the files haven't been extracted to disk yet. Was considering sharing some logic between installation and signature validation, but since referred difference decided to do like this. Thoughts?

Reply:
I think it's fine to keep separate for now. Always time in the future to tidy up and refactor if necessary